src/Components/Alert/Components/Filters/NotificationPreferences.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  Box,
+  Checkbox,
+  Message,
+  Skeleton,
+  SkeletonBox,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { RouterLink } from "System/Router/RouterLink"
+import { FC, useEffect } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { NotificationPreferences_viewer$data } from "__generated__/NotificationPreferences_viewer.graphql"
+import { NotificationPreferencesQuery } from "__generated__/NotificationPreferencesQuery.graphql"
+import { AlertFormikValues } from "Components/Alert/Components/Steps/Details"
+import { useFormikContext } from "formik"
+
+interface NotificationPreferencesProps {
+  viewer?: NotificationPreferences_viewer$data
+}
+
+export const NotificationPreferences: FC<NotificationPreferencesProps> = ({
+  viewer,
+}) => {
+  const { state } = useAlertContext()
+
+  const { setFieldValue, values } = useFormikContext<AlertFormikValues>()
+
+  const areCustomAlertsEmailNotificationsEnabled = viewer?.notificationPreferences?.some(
+    preference => {
+      return (
+        preference.channel === "email" &&
+        preference.name === "custom_alerts" &&
+        preference.status === "SUBSCRIBED"
+      )
+    }
+  )
+
+  useEffect(() => {
+    setFieldValue("email", areCustomAlertsEmailNotificationsEnabled)
```

Comment:
Updating the `email` value in an effect because I cannot set the default value from this component.